README.md

```diff
@@ -1,10 +1,16 @@
 # AEProject
 
-**AEProject** is an aeternity framework which helps with setting up an
-project.
+**AEProject** is an aeternity framework which helps with setting up an project.
+The framework makes the development of smart contracts in the aeternity network pretty easy. It provides commads
```

Comment:
`commands`. Try running the whole thing through spellchecker as I am not great at that